package.json

```
@@ -34,7 +34,7 @@
     "test:e2e:all": "cross-env E2E_TESTS=true start-server-and-test start http://localhost:5678/favicon.ico 'cypress run --headless'"
   },
   "dependencies": {
-    "n8n": "*"
+    "n8n": "^0.210.1"
```

Please change this to 
```suggestion
    "n8n": "workspace:*"
```

---

Just did but not sure what I messed up all the back-end tests are failing with an SQLite error.